test/test_newton_trust_region.jl

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env julia
```

Comment:
Hey Ryan, That'd be great if you'd add those tests whenever you have the
chance.

What's left to do to merge it with Optim.jl?



On Thu, Jun 23, 2016 at 4:51 PM Ryan <notifications@github.com> wrote:

> In test/test_newton_trust_region.jl
> <https://github.com/jeff-regier/Celeste.jl/pull/234#discussion_r68335123>:
>
> > @@ -0,0 +1,31 @@
> > +#!/usr/bin/env julia
>
> Thanks for doing all this!
>
> There's no reason you would know, but I wrote a unit test that's in the
> trust_region branch of my fork of Optim.jl. You may want to give it a try,
> or LMK if you would like me to do it. I can get to it after ICML.
>
>
> https://github.com/rgiordan/Optim.jl/blob/trust_region/test/newton_trust_region.jl
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/jeff-regier/Celeste.jl/pull/234/files/61f6865ff83c67ac04b8d3176a6f869ff8cc5539#r68335123>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AIieC7pg3vNcd0obPUleG7toxmBVjn82ks5qOxvigaJpZM4I9Rd6>
> .
>